back-office/customer-service/customer-service/dashboard.md

```diff
@@ -0,0 +1,370 @@
+# SPECIFICATIONS - CUSTOMERS SERVICE > CUSTOMERS SERVICE PAGE
+
+## MOCKUPS
+
+https://www.figma.com/proto/axasD3Kyg52okWHqvWPN1I/PrestaShop---Customer-Service-(migration)?node-id=12%3A4083&scaling=min-zoom
+
+https://www.figma.com/proto/axasD3Kyg52okWHqvWPN1I/PrestaShop---Customer-Service-(migration)?node-id=12%3A5267&scaling=min-zoom
+
+https://www.figma.com/proto/axasD3Kyg52okWHqvWPN1I/PrestaShop---Customer-Service-(migration)?node-id=12%3A9835&scaling=min-zoom
+
+https://www.figma.com/proto/axasD3Kyg52okWHqvWPN1I/PrestaShop---Customer-Service-(migration)?node-id=12%3A8042&scaling=min-zoom
+
+https://www.figma.com/proto/axasD3Kyg52okWHqvWPN1I/PrestaShop---Customer-Service-(migration)?node-id=12%3A11958&scaling=min-zoom
+
+## MESSAGES NOTIFICATIONS
+
+When a new message is sent by a customer or a visitor, then it's displayed in the notifications (bell) in the header.
+
+If you click on it, the thread is opened (view page) and all the messages notifications are removed.
```

Comment:
Maybe add the info that the notification is removed for this user only (but since it's the general behavior for notifications, it's might not be useful).

Reply:
Nice catch, thank you @Robin-Fischer-PS :)